Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15 as builder
+
+COPY go.mod go.sum /
+RUN go mod download
+
+ADD . /code
+WORKDIR /code
+RUN make build
+
+FROM scratch
+COPY --from=builder /code/bin/pbnj-linux-amd64 /pbnj-linux-amd64
+ENTRYPOINT ["/pbnj-linux-amd64", "server"]
```

Comment:
Lets throw this up at the top, ENTRYPOINT is really just a container runtime metadata data field and I think it looks nice to have:

```dockerfile
FROM

EXPOSE
ENTRYPOINT
CMD

...
```

Reply:
is this organization from some standard or doc somewhere? I typically like to follow others who do more of a logical mind mapping for organizing from top to bottom. for example https://github.com/jessfraz/dockerfiles/blob/master/cfssl/Dockerfile

Reply:
Yeah I feel the same as @jacobweinstock when I look at a dockerfile I typically see entrypoint and cmd set at the bottom. It also just feels more proper that at the end you would define those.

Reply:
Just matches all the other repos in tinkerbell org, likely originating from me :D. My reasoning for it is its container metadata and not anything that changes the files on the layers. I like to have all the metadata up front and then can see how layers and files are going to be affected. Probably also busted the cache back in the day and likely not an issue if using buildx now adays.

Having one way of doing things (even if wrong) is better than multiple, so if we go with at the end (sounds fine to me), can you open up PRs for the other repos too please.

Reply:
sure